meson.build

```diff
@@ -170,6 +170,11 @@ if true
     src_files += 'src/modules/wlr/taskbar.cpp'
 endif
 
+if true
+    add_project_arguments('-DHAVE_RIVER', language: 'cpp')
+    src_files += 'src/modules/river/tags.cpp'
+endif
```

Comment:
this `if true` and the existence of the `HAVE_RIVER` define are pure cargo-culting, I just copied what is done for the `wlr/taskbar` module. The `river/tags` module does not introduce any external dependencies, it only requires `wayland-scanner` and the protocol xml which I have included in this PR.